lib/gli.rb

```diff
@@ -24,10 +24,20 @@
 
 module GLI
   include GLI::App
+  @@use_hyphenized_form = false
```

Comment:
I'm almost done with this. Just having a hard time figuring out what's the best way to make sure that CommandLineOption can see the dasherize_underscore_options flag.